WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -155,6 +157,18 @@ private extension SupportTableViewController {
             rows: helpSectionRows,
             footerText: LocalizedText.helpFooter)
 
+        // Community Forums Section
+        var communityForumsSectionRows = [ImmuTableRow]()
+        communityForumsSectionRows.append(SupportForumRow(title: LocalizedText.wpForumPrompt,
+                                                          accessibilityHint: "",
+                                                          action: nil,
+                                                          accessibilityIdentifier: ""))
+        communityForumsSectionRows.append(SupportForumButtonRow(title: LocalizedText.visitWpForumsButton,
+                                                                accessibilityHint: "",
```

Comment:
I think we could add a hint explaining this button opens a mobile forum website on a browser. How do you see it?

Reply:
@staskus could you please check you have the latest / were reviewing the latest?  I did add a hint [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/2b1c9998213bb4a3aaec3c2f99245dff139e4882/WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift#L166) thanks!